src/main/java/org/truffleruby/options/CommandLineParser.java

```diff
@@ -156,8 +156,8 @@ private void processArgv() {
             }
         }
         // Remaining arguments are for the script itself
-        arglist.addAll(Arrays.asList(OptionsCatalog.ARGUMENTS.checkValue(config.getOptions().get(OptionsCatalog.ARGUMENTS.getName()))));
-        config.getOptions().put(OptionsCatalog.ARGUMENTS.getName(), arglist.toArray(new String[arglist.size()]));
+        //arglist.addAll(Arrays.asList(OptionsCatalog.ARGUMENTS.checkValue(config.getOptions().get(OptionsCatalog.ARGUMENTS.getName()))));
```

Comment:
I'd rather just remove the code if it's not working anymore.
This used to allow setting args as a Java property and concatenate them with CLI args.
I don't know of any use-case.